### ISSUE TITLE
Duplicate metrics

### ISSUE THREAD

**Issue by Eloo** [micrometer-metrics/micrometer#53]
It seems that if many request came for the very first time that duplicate counters will be registered which leads to duplicate metrics in exporting and therefore overwriting metrics.

This will look like this.
```
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.5"} 1.436569357
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.95"} 2.810563785
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.5"} NaN
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.95"} NaN
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.5"} NaN
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.95"} NaN
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.5"} NaN
http_server_requests{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}",quantile="0.95"} NaN
http_server_requests_sum{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}"} 0
http_server_requests_count{exception="None",instance="192.168.110.111",instanceType="local",job="MVG",method="DELETE",profile="dev",status="500",uri="v1/vehicles/{vin}"} 0
```
It seems that the dynamic creation of the endpoint metric should be synchronized over all threads?

Best regards,
Eloo

**Comment by jkschneider** [comment]
Interesting! Thanks for the diagnosis.

**Comment by Eloo** [comment]
the issue occured while running with gatling (atOnceUsers(500)) against my restapi

**Comment by Eloo** [comment]
can confirm that this is fixed in 0.7.0.RELEASE
